.travis.yml

```diff
@@ -47,6 +53,15 @@ matrix:
       env: TOXENV=individual_coverage
     - python: 3.5
       env: TOXENV=docs
+    - language: node_js
+      node_js:
+          - "6"
+          - "iojs"
+      install: cd $MITMWEB_DIR && npm install
```

Comment:
Can we use yarn? I'd like to have a bit more consistency :) 
https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn